core/src/main/java/org/mule/DefaultMuleEvent.java

```diff
@@ -83,7 +86,7 @@
     /** Mutable MuleEvent state **/
     private boolean stopFurtherProcessing = false;
     private int timeout = TIMEOUT_NOT_SET_VALUE;
-    private transient ResponseOutputStream outputStream;
```

Comment:
There is only one field defined as ResponseOutputStream in all mule code. So I guess, the rest of the usages are only in order to fill that field. Now that you are removing that usage, I guess is safe to deprecate it, but it could be in a separated issue